pixart.js

```diff
@@ -1 +1,18 @@
+$(document).ready(function() {
+  $('button').click(function(evt) {
+    evt.preventDefault()
+    console.log("Clicked")
+  });
+});
+$('#Set Color').bind('keypress', function(e) {
```

Comment:
An ID can't contain spaces, so this may have been what was breaking the app. IDs should also usually be lowercase. Maybe `$("#set_color")` instead?